packages/design-system/src/components/WIP/ErrorState/ErrorState.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { ReactElement } from 'react';
+
+import { ButtonPrimary } from '../../Button';
+import { ButtonPrimaryPropsType } from '../../Button/variations/ButtonPrimary';
+import { StackVertical } from '../../Stack';
+
+import ErrorIllustration from './illutstrations/ErrorIllustration';
+
+import styles from './ErrorState.module.scss';
+
+type ErrorStatePropTypes = {
+	title: string;
+	description: string;
+	action?: Omit<ButtonPrimaryPropsType<'M'>, 'size'>;
+	link?: ReactElement | string;
```

Comment:
```suggestion
	link?: ReactElement | LinkProps;
```
We must try to ensure that there is a link in there. `string` would be useless.

By checking if the prop is a function or an object, you can either display the `<Link />` component or the provided node.